### ISSUE TITLE
Does not sync to server

### ISSUE THREAD

**Issue by Gigor911** [nek4life/sfcc-studio#1]
Hi, we've been using your DW plugin for a while. Today I tried using the new one, but was not able to upload files. Can you suggest please if there were any changes in upload process? Should cartridges be marked as sources root? And where can I find connection setting after project is created? Should upload version be an active version?
Seems like I have filled all the fields correctly, but clicking clean cartridges does nothing.

**Comment by nek4life** [comment]
@Gigor911 Sorry I need to work on writing the docs to setting the project up. The cartridge path settings are in the project settings. I've moved everything from facet configuration to project settings. When opening the settings search for Commerce Cloud. There is a UI to select all your cartridges to add the cartridge roots. Once you do this the server will know what paths to watch. 

I would have added this to the wizard, but if you're creating a new project you would not have any cartridges setup yet. Let me know if this helps!

**Comment by Gigor911** [comment]
That helps, thank you so much.

**Comment by nek4life** [comment]
No problem! Let me know if that fixes your issue. I'll keep this ticket open to remind me to write some better getting started docs.

**Comment by Gigor911** [comment]
Yes, it is working like a charm. Thank you for this plugin. It really helps, cause I hate eclipse and VSCode does not provide all the tools I need. Btw, are you planning to work on debugging?

**Comment by nek4life** [comment]
That's pretty much the same reason I built this and the original plugin in the first place. 😄 

I started looking into the debugging API a little bit and plugin hooks to adding a custom debugger into Intellij. I'd definitely love to add it to the plugin, but it looks like a bit of a beast. I'll keep poking around and see if I can get something working eventually.

The good news is that the plugin is a lot easier to setup and get working on now that it uses the gradle build system so maybe someone smarter than me can take a crack at it as well... 💪
